src/python/pants/backend/jvm/tasks/jvm_compile/class_not_found_error_patterns.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import re
+
+
+_CLASS_NOT_FOUND_ERROR_PATTERNS = [
```

Comment:
Would it be crazy to have these in a pants `list` option, so that users could add/remove them via configuration?

Also, having these as module-level constants will cause the regexes to be compiled during pants' startup... if you're not going to move them to an option, you should still make them members of a class so that they will be compiled only if they will be used.